R/Aligning_FT_QTOF_SQL.R

```diff
@@ -0,0 +1,84 @@
+Aligning_FT_QTOF_SQL <- function(
+    FT_path, QTOF_path, expnr.ft, expnr.syn, 
```

Comment:
would it not be easier for the user to use consistent formatting and naming? e.g. either use `_` or `.` in parameter names, not both. also, it could be confusing to use QTOF and syn for the same thing. might maybe be more consistent to call them e.g. `FT_path`, `QTOF_path`, `FT_expnr` and `QTOF_expnr`?